pipeline-templates/install-node.yaml

```diff
@@ -1,5 +1,5 @@
 steps:
-- task: NodeTool@0
+- task:  UseNode@1
   inputs:
-    versionSpec: '16.10.0'
+    version: '18.x'

```

Comment:
Is this an open version/minimum version? Any risk with installing different versions of 18.x and getting different results in pipelines?

Reply:
IMO no risk - this is the runtime used to build/package the tool, but not the runtime used by end users when they run our extension. The user-applicable nodejs runtime is controlled entirely by VSCode and kept consistent/updated by them.

Reply:
That's a good question and I agree with Chet. There is minimal risk. 18 is also LTS and pretty old at this point (think .NET 6)